src/IncrementalFetchingHelper.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\AzureStorageTableExtractor;
+
+use Keboola\AzureStorageTableExtractor\Configuration\Config;
+use Keboola\AzureStorageTableExtractor\Exception\UndefinedValueException;
+use Keboola\AzureStorageTableExtractor\Exception\UserException;
+use Keboola\Component\JsonHelper;
+use MicrosoftAzure\Storage\Table\Models\EdmType;
+
+class IncrementalFetchingHelper
```

Comment:
Incremental fetching sa tu tiez musi riesit zlozito:
- Databaza nepodporuje `ORDER BY`, preto vzdy musim porovnavat hodnoty a ukladat maximum.
- V `QueryFactory` musim uviest spravny typ polozky, inak mi to nevrati ziadne data.
- Preto sa okrem hodnoty uklada aj typ a kontroluje sa ci je pre vsetky riadky rovnaky.
- To vsetko so vyclenil do tejto osobitnej triedy.

Reply:
Vid testy.